helm/charts/infra/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: infra
 description: A Helm chart for Infra
 type: application
-version: 0.14.4
+version: 0.20.0
 appVersion: 0.14.4
```

Comment:
Does the appVersion need to change too?

Reply:
It does not. The default Docker image for this Helm release is still 0.14.4

Reply:
I'm still worried about us deviating here, since they've been in lockstep before. I'm just worried that someone will get confused.

Reply:
Differing chart and app versions is common, probably more common than same chart and app versions. It provides us flexibility going forward to make changes to the helm chart independently of the app which may move at a faster or slower pace.